lib/core-integration/src/Test/Integration/Scenario/API/Shelley/StakePools.hs

```diff
@@ -416,68 +428,108 @@ spec = do
             , expectErrorMessage $ errMsg403DelegationFee fee
             ]
 
-    it "STAKE_POOLS_LIST_01 - List stake pools" $ \ctx -> do
-        eventually "Listing stake pools shows expected information" $ do
-            r <- request @[ApiStakePool] ctx
-                (Link.listStakePools arbitraryStake) Default Empty
-            expectResponseCode HTTP.status200 r
-            verify r
-                [ expectListSize 3
-
--- Pending a mock metadata registry
---                , expectListField 0
---                    #metadata ((`shouldBe` Just "Genesis Pool") . fmap (view #name . getApiT))
---                , expectListField 1
---                    #metadata ((`shouldBe` Just "Genesis Pool") . fmap (view #name . getApiT))
---                , expectListField 2
---                    #metadata ((`shouldBe` Just "Genesis Pool") . fmap (view #name . getApiT))
-
-                , expectListField 0
-                    #cost (`shouldBe` (Quantity 0))
-                , expectListField 1
-                    #cost (`shouldBe` (Quantity 0))
-                , expectListField 2
-                    #cost (`shouldBe` (Quantity 0))
-
-                , expectListField 0
-                    #margin (`shouldBe` (Quantity minBound))
-                , expectListField 1
-                    #margin (`shouldBe` (Quantity minBound))
-                , expectListField 2
-                    #margin (`shouldBe` (Quantity minBound))
-
--- Pending stake pools producing blocks in our setup,
--- AND pending keeping track of block producions
---                , expectListField 0
---                    (#metrics . #producedBlocks) (.>= Quantity 0)
---                , expectListField 1
---                    (#metrics . #producedBlocks) (.>= Quantity 0)
---                , expectListField 2
---                    (#metrics . #producedBlocks) (.>= Quantity 0)
---
---                , expectListField 0
---                    (#metrics . #nonMyopicMemberRewards) (.>= Quantity 0)
---                , expectListField 1
---                    (#metrics . #nonMyopicMemberRewards) (.>= Quantity 0)
---                , expectListField 2
---                    (#metrics . #nonMyopicMemberRewards) (.>= Quantity 0)
-
-                , expectListField 0
-                    (#metrics . #saturation) (.>= 0)
-                , expectListField 1
-                    (#metrics . #saturation) (.>= 0)
-                , expectListField 2
-                    (#metrics . #saturation) (.>= 0)
-                ]
+    let fixTypeInference = True `shouldBe` True
+    let listPools ctx = request @[ApiStakePool] @IO ctx (Link.listStakePools arbitraryStake) Default Empty
+    describe "STAKE_POOLS_LIST_01 - List stake pools" $ do
+            it "immediately has non-zero saturation & stake" $ \ctx -> do
```

Comment:
neat-picking but why the double indentation here?

Reply:
A remnant, fixed.